src/app/containers/MediaPlayer/Metadata.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import Helmet from 'react-helmet';
-import videoMetadata from './helpers/metadata';
+import { shape } from 'prop-types';
+import mediaPlayerMetadata from './helpers/metadata';
 
-const Metadata = aresMediaBlock => {
-  const metadata = videoMetadata(aresMediaBlock);
+const Metadata = ({ aresMediaBlock }) => {
```

Comment:
@jamesdonoh related this [comment](https://github.com/bbc/simorgh/pull/4339/files#r337988177),
the destructuring here ensured that the `aresMediaBlock` prop was appropriately  passed down as opposed to how it was before when it wasn't since it wasn't been called as a function [here](https://github.com/bbc/simorgh/blob/latest/src/app/containers/MediaPlayer/index.jsx#L61)